easybuild/easyconfigs/b/Beast/Beast-2.6.7-GCC-10.3.0-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Tarball'
+
+name = 'Beast'
+version = '2.6.7'
+local_cuda_ver = '11.3.1'
+versionsuffix = '-CUDA-%s' % local_cuda_ver
+
+homepage = 'http://beast2.org'
+description = """ BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
+ sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
+ strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
+ but is also a framework for testing evolutionary hypotheses without conditioning on a single 
+ tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
+ proportional to its posterior probability. """
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/CompEvol/beast2/releases/download/v%(version)s/']
+sources = ['BEAST.v%(version)s.Linux.tgz']
+checksums = ['05dcc619c2e10163f2c1089ec66149f6e53ec5a0583cd2cb8ffdccbbdb1d8183']
+
+dependencies = [
+    # this is not mandatory but beagle-lib is recommended by developers
+    # beagle-lib will also load the required java dependency
+    # if you remove this you should add the java dependency
+    ('beagle-lib', '3.1.2', versionsuffix),
+]
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('pkg-config', '0.29.2'),
+]
```

Comment:
not sure that build dependencies make a lot of sense when using the `Tarball` easyblock, since you're not actually doing any building?